src/Persistence_SQL.php

```diff
@@ -590,7 +590,11 @@ public function action($m, $type, $args = [])
      */
     public function tryLoad(Model $m, $id)
     {
-        $load = $this->action($m, 'select');
+        if (!$m->id_field) {
+            throw new Exception(['Unable to load field by "id" when Model->id_field is not defined.', 'id'=>$id]);
+        }
+
+        $load = $m->action('select');
```

Comment:
No need to call Model->action here. It's slightly faster (one function call less) to call persistence->action instead as it was before.